mpas_analysis/ocean/meridional_heat_transport.py

```diff
@@ -105,6 +100,22 @@ def meridional_heat_transport(config):  # {{{
                       'older version of MPAS-O that did not support this flag.  Assuming .false.')
         mhtAnalysisMemberFlag = '.false.'
 
+    # Read in depth and MHT latitude points
+    # Latitude is from binBoundaryMerHeatTrans written in mpaso.hist.am.meridionalHeatTransport.*.nc
+    # Depth is from refZMid, also in mpaso.hist.am.meridionalHeatTransport.*.nc
+    try:
+        mhtFile = runStreams.readpath('meridionalHeatTransportOutput')[0]
```

Comment:
I believe this should be taken from `historyStreams`, not `runStreams`.  This only matters for runs where short-term archiving is on, but `runStreams` are for things like restart files that stay in the run directory.  Am I correct in thinking that the output from the `meridionalHeatTransportOutput` will go to short-term archiving?